docs/annihilation.rst

```diff
@@ -136,4 +147,51 @@ Any force that is not in one of the above-mentioned classes will cause an except
 - CMMotionRemover
 
 
+Additional Features
+^^^^^^^^^^^^^^^^^^^
+In addition to creating a hybrid topology dependent on a set of `lambda_force` functions, the HybridTopologyFactory can also take advantage of OpenMM's automatic differentiation to provide dU/dlambda. In order to use this
+functionality, you must provide alchemical functions to the constructor of HybridTopologyFactory. These functions are described below.
+
+Basic Usage of HybridTopologyFactory
+------------------------------------
+
+Basic Construction of Factory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In order to use the HybridTopologyFactory, several objects are necessary:
+
+- :py:class:`perses.rjmc.topology_proposal.TopologyProposal` is a container class which holds so-called new and old systems, topologies, and respective atom maps. It can be created manually using the constructor, or can be obtained through a subcalss of :py:class:`perses.rjmc.topology_proposal.ProposalEngine`.
+
+- You need to supply current positions and new positions for the old and new systems, respectively. It is assumed that the atoms which are mapped will have the same positions. If you only have positions for the old system, you can generate (decent) positions for the new system using :py:class:`perses.rjmc.geometry.FFAllAngleGeometryEngine`. You should minimize the hybrid topology that results after this to avoid clashes, however.
+
+- The dispersion correct is calculated for the sterics force if `use_dispersion_correction` is set to `True`. It is false by default; whether to enable this depends on what sort of calculation you are doing:
```

Comment:
I think restructured text uses two backquotes to quote code:
http://www.sphinx-doc.org/en/stable/rest.html